source/common/router/config_impl.h

```diff
@@ -13,6 +13,7 @@
 #include "envoy/api/v2/rds.pb.h"
 #include "envoy/api/v2/route/route.pb.h"
 #include "envoy/common/optional.h"
```

Comment:
Do you want to remove the old one in the same sweep?

Reply:
Well, it's still a work in progress PR. I don't mind doing that either. Let me know if you want me to do that.

Reply:
Sure, let's just do one giant PR, it should be fairly easy to review, as the transforms should all be mechanical. Thanks.